docs/functions.rst

```diff
@@ -1166,6 +1166,60 @@ Example:
       args:
         snapshotID: "{{ .ArtifactsIn.backupInfo.KeyValue.snapshotID }}"
 
+
+KubeOps
+-------
+
+This function is used to manage Kubernetes resources. We can use it to
```

Comment:
instead of manage can we just use create/delete in the first statement itself?

Reply:
I mentioned manage because more operations will be added to this function

Reply:
If mentioning create/delete specifically sounds more correct then I'll make this change.